units/systemd-sysupdate-reboot.timer

```diff
@@ -0,0 +1,20 @@
+#  SPDX-License-Identifier: LGPL-2.1-or-later
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=Reboot Automatically After System Update
+Documentation=man:systemd-sysupdate-reboot.service(8)
+ConditionVirtualization=!container
+
+[Timer]
+OnCalendar=4:10
```

Comment:
You have both options. If you specify `systemd-update now --reboot` we'll immediately reboot once the update is done. If you don't do that and just call `systemd-update now` then we'll not reboot immediately. In that case it might make sense to enable this timer unit, which will reboot then in the middle of the night, if there's a newer version available. which policy people pick is up to them really.